content/exchange/artifacts/HollowsHunter.yaml

```diff
@@ -0,0 +1,28 @@
+name: Custom.Windows.Memory.HollowsHunter
+description: |
+   Use hollows_hunter to detect suspicious process injections.
+   
+   Upload any findings to the server, including process dumps.
+tools:
+ - name: hollows_hunter
+   url: https://github.com/hasherezade/hollows_hunter/releases/download/v0.2.7.1/hollows_hunter64.exe
```

Comment:
Shall we update to the latest? 

You can change this to the github to do it automatically

https://github.com/Velocidex/velociraptor/blob/8e060d40aa4f28946059af689a34486d5eb0ef1f/artifacts/definitions/Server/Utils/CreateCollector.yaml#L13